github/scim.go

```diff
@@ -29,6 +33,8 @@ type SCIMUserAttributes struct {
 	ExternalID  *string          `json:"externalId,omitempty"`  // (Optional.)
 	Groups      []string         `json:"groups,omitempty"`      // (Optional.)
 	Active      *bool            `json:"active,omitempty"`      // (Optional.)
+	ID          *string          `json:"id,omitempty"`          // Only populated as a result of calling ListSCIMProvisionedIdentitiesOptions or GetSCIMProvisioningInfoForUser
+	Meta        *SCIMMeta        `json:"meta,omitempty"`        // Only populated as a result of calling ListSCIMProvisionedIdentitiesOptions or GetSCIMProvisioningInfoForUser
```

Comment:
Let's put the long comment after a blank line for the two last fields (with a colon at the end), like this:

```suggestion

	// Only populated as a result of calling ListSCIMProvisionedIdentitiesOptions or GetSCIMProvisioningInfoForUser:
	ID		*string		`json:"id,omitempty"`
	Meta	*SCIMMeta	`json:"meta,omitempty"`        
```